src/main/java/uk/gov/hmcts/reform/unspec/aspect/NoOnGoingBusinessProcessAspect.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.unspec.aspect;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.unspec.callback.CallbackParams;
+import uk.gov.hmcts.reform.unspec.callback.CaseEvent;
+import uk.gov.hmcts.reform.unspec.helpers.CaseDetailsConverter;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+
+import java.util.List;
+
+import static java.lang.String.format;
+import static uk.gov.hmcts.reform.unspec.enums.BusinessProcessStatus.FINISHED;
+
+@Slf4j
+@Aspect
+@Component
+@RequiredArgsConstructor
+public class NoOnGoingBusinessProcessAspect {
+
+    public static final String ERROR_MESSAGE = "There is a technical issue causing a delay. "
```

Comment:
Should we include that the error is caused by ongoing business process in message to the user?

Reply:
This message is provided by the content team in this contect ticket: CMC-800